lib/include/rmb/command/SuicidalCommand.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include <frc2/command/CommandBase.h>
+#include <frc2/command/CommandHelper.h>
+#include <frc2/command/Subsystem.h>
+#include <frc2/command/SequentialCommandGroup.h>
+#include <frc2/command/InstantCommand.h>
+
+namespace rmb {
+  /**
+   * Scheduels a command that will delete itself after `End()` is called. This 
+   * is Achived by wraping the command in an internal `SuicidalCommand` that 
+   * takes ownership of the command. This `Sucidal command is schedueld and
+   * will delete itself and the inital command with it once it is ended.
+   * 
+   * @param command Command to be schedueled (construt in place).
+   **/
+  void scheduelSuicidalCommand(std::unique_ptr<frc2::Command> command);
```

Comment:
Shoud this be a `frc2::Command&&` instead of a `std::unique_ptr<frc2::Command>`?

Reply:
well actually, this might work. But I think an rvalue reference looks better